spk/sickchill/src/requirements-crossenv.txt

```diff
@@ -1,10 +1,10 @@
-cffi==1.15.0
-cryptography==3.3.2
-greenlet==1.1.2
-lxml==4.6.4
-MarkupSafe==2.0.1
-msgpack==1.0.2
-PyNaCl==1.4.0
-SQLAlchemy==1.4.26
-tornado==6.1
-wrapt==1.13.3
+cffi==1.15.1
+cryptography==38.0.1
```

Comment:
Now I noticed this, so you're building both `py310-py310` and a `py36-abi3` wheels.  Therefore you end-up having two versions of the same wheel in the `work-<arch>-<ver>/wheelhouse/cryptography*.whl`.  Only one is needed.

Can you check which version was used on your tested installations as you should only be keeping one or the other.